cloudbuild.yaml

```diff
@@ -374,6 +374,23 @@ steps:
     dir: cmd/sdk-server/bin
     args: [-m, cp, '*.zip', gs://agones-artifacts/sdk-server]
 
+  #
+  # Cleanup services "preview", "development", "default"
+  #
+  - name: gcr.io/cloud-builders/gcloud
+    id: cleanup-services
+    waitFor: [htmltest-save-cache, rust-build-save-cache, tests, build-images]
+    entrypoint: bash
+    args:
+      - -c
+      - |
+        set -e
+        set -o pipefail
+        for service in preview development default; do
+        echo "Cleaning up $service service..."
+        gcloud app versions list --service=$service --filter="traffic_split<1" --sort-by=~last_deployed_time.datetime --format="table[no-heading](version.id)"
```

Comment:
two things:
* please add the `tail` from the original so we can see what would be deleted. As it is now it's just outputting everything (in sorted order).
* please indent 390/391